Source/WebKit/WebProcess/glib/SystemSettingsManager.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 Purism SPC
+ * Copyright (C) 2024 Igalia S.L.
```

Comment:
Shouldn't we keep the Purism copyright line along with the added Igalia one?

Reply:
Oops, yes.